src/api/abstract-tree.h

```diff
@@ -113,6 +113,7 @@ struct yfa_funcdecl {
     struct yf_ast_node * body; /* The function body */
     /* The parameter scope */
     struct yfs_symtab * param_scope;
+    bool extc; /* Whether the function declaration is extc */
```

Comment:
Would this be a good place for a union of struct function and struct external c function?

Reply:
Given that the only difference between a normal function and an extc function is a single boolean flag, I'm not sure another way would really be more efficient.

Reply:
That makes sense, yea